script/windows/14.0/update/WindowsUpdateCheck.php

```diff
@@ -1,15 +1,14 @@
 <?php
+  namespace Keyman\Site\com\keyman\api;
+
   require_once __DIR__ . '/../../../../tools/util.php';
 
-  class OnlineUpdate {
+  class WindowsUpdateCheck {
     const MSI_REGEX = '/^keyman(desktop)?\.msi$/';
     const BOOTSTRAP_REGEX = '/^setup\.exe$/';
     const BUNDLE_REGEX = '/^keyman(desktop)?-.+\.exe/';
```

Comment:
Will the name change from "Keyman Desktop" --> "Keyman for Windows" mean the installer names change too? 
e.g. regex `keyman(windows)`?

Reply:
I am proposing that the new filename for a bundled install be `keyman-a.b.c.exe`.